safe-global/safe-wallet-web
# Fix: add loading icon to Sign/Execute buttons

## What it solves
When signing/executing for social login, since no popup shows up the user stays in the form for a few seconds not knowing if something is happening. I added a loading icon in the sign/execute button when clicked. The loading icon replaces the text to not messup with the buttons width

## How this PR fixes it
Adds the icon when the button is grayed out (a tx is executing or being signed)

## How to test it
The feature affects signing, execution and execution in bulk
Go to a form as an owner and click the sign/execute/submit, you sould see the loading icon replacing the text
Go to a form an disconnect. The button should NOT show the loading icon
Go to a form and switch to a non-owner. The button should NOT show the loading icon
Go to a fully sign tx with a non-owner. You SHOULD see the loading icon

## Screenshots
Should look like this:
![image](https://github.com/safe-global/safe-wallet-web/assets/4503659/07da1a6e-0706-4b54-b9a8-2723a7f2b751)


## Checklist
